mappings/net/minecraft/block/RedstoneTorchBlock.mapping

```diff
@@ -1,9 +1,25 @@
 CLASS bqr net/minecraft/block/RedstoneTorchBlock
+	CLASS bqr$a BurnoutEntry
+		FIELD a pos Lev;
+		FIELD b time J
+		METHOD <init> (Lev;J)V
+			ARG 1 pos
+			ARG 2 time
 	FIELD a LIT Lbwb;
+	FIELD b BURNOUT_MAP Ljava/util/Map;
 	METHOD <init> (Lbml$c;)V
 		ARG 1 settings
-	METHOD a (Lbhh;Lev;Lbvj;)Z
+	METHOD a shouldUnpower (Lbhh;Lev;Lbvj;)Z
```

Comment:
Unpower? Not burn out?

Reply:
Yes, this is related to unpowering the torch when it receives a redstone signal. It's overridden in `RedstoneTorchWallBlock` e.g.

Reply:
`shouldInvert`? Inversion is how this is described in the redstone community?